versions/3.0.md

```diff
@@ -248,6 +246,21 @@ name: Apache 2.0
 url: http://www.apache.org/licenses/LICENSE-2.0.html
 ```
 
+#### <a name="componentsObject"></a>Components Object
+
+Holds a set of schemas for different aspects of the OAS.  The intention is to put reusable components into a single location, allowing reuse from this and other OAS documents.
+
+##### Fixed Fields
+
+Field Pattern | Type | Description
+---|:---:|---
+<a name="definitionsObject"></a> | [Definitions Object](#definitionsObject) | A hash containing payload definitions for the specification.
+<a name="responsesDefinitionsObject"></a> | Responses Definitions Object | Reusable responses objects.
+<a name="parametersDefinitionsObject"></a> | [Parameters Definitions Object](#parametersDefinitionsObject) | An object to hold parameters to be reused across operations. Parameter definitions can be referenced to the ones defined here.
+<a name="responseHeadersDefinitionsObject"></a> | [Response Headers Definitions Object](#responseHeadersDefinitionsObject) | Response headers to reuse across the specification.
+<a name="securityDefinitionsObject"></a> | [Security Definitions Object](#securityDefinitionsObject) | Security definitions to reuse across the specification.
```

Comment:
Here is still something missing – in the first column we need the actual field names of this components object (like `definitions`, `responses`, `parameters`, ...), and different `name` attributes (they currently are the same as for the objects linked in the second column, which will break stuff). I guess something like `componentsDefinitions`, `componentsResponses`, ... would do for the names.

I only noted this after the merge (trying to rebase my #650), sorry.